Detectors/TOF/calibration/testWorkflow/LHCClockCalibratorSpec.h

```diff
@@ -54,39 +57,69 @@ class LHCClockCalibDevice : public o2::framework::Task
     mCalibrator->setSlotLength(slotL);
     mCalibrator->setMaxSlotsDelay(delay);
 
-    // calibration objects set to zero
-    mPhase.addLHCphase(0, 0);
-    mPhase.addLHCphase(2000000000, 0);
-
-    TFile* fsleewing = TFile::Open("localTimeSlewing.root");
-    if (fsleewing) {
-      TimeSlewing* ob = (TimeSlewing*)fsleewing->Get("ccdb_object");
-      mTimeSlewing = *ob;
-      return;
-    }
-
-    for (int ich = 0; ich < TimeSlewing::NCHANNELS; ich++) {
-      mTimeSlewing.addTimeSlewingInfo(ich, 0, 0);
-      int sector = ich / TimeSlewing::NCHANNELXSECTOR;
-      int channelInSector = ich % TimeSlewing::NCHANNELXSECTOR;
-      mTimeSlewing.setFractionUnderPeak(sector, channelInSector, 1);
+    if (!mUseCCDB) {
+      // calibration objects set to zero
+      mPhase.addLHCphase(0, 0);
+      mPhase.addLHCphase(o2::ccdb::CcdbObjectInfo::INFINITE_TIMESTAMP_SECONDS, 0);
+      if (gSystem->AccessPathName("localTimeSlewing.root") == false) {
```

Comment:
where this local file supposed to come from?

Reply:
It was used for local testing only.